RiotShareExtension/target.yml

```diff
@@ -38,8 +38,8 @@ targets:
     sources:
     - path: .
       excludes:
-      - "Shared/View/ShareRoomsRoomCell.*"
-      - "Shared/View/ShareRoomsDiscussionCell.*"
+      - "**/ShareRoomsRoomCell.*"
+      - "**/ShareRoomsDiscussionCell.*"
```

Comment:
@phlniji I think we can remove definitely this 2 classes ("ShareRooms...") from the project, by deleting the source code from Tchap repo. Do you agree?

Reply:
Yes, they are now useless. I remove them.